access-common/src/main/java/edu/unc/lib/dl/ui/service/FedoraContentService.java

```diff
@@ -176,9 +176,11 @@ private void streamData(String simplepid, Datastream datastream, String filename
 					String mimeType;
 					try {
 						mimeType = method.getResponseHeader("content-type").getValue();
-						if (mimeType == null || "application/octet-stream".equals(mimeType)) {
+						if (mimeType == null || "application/octet-stream".equals(mimeType) || "application/download".equals(mimeType) || "application/x-download".equals(mimeType)) {
 							if ("mp3".equals(datastream.getExtension())) {
 								mimeType = "audio/mpeg";
+							} else if ("pdf".equals(datastream.getExtension())) {
```

Comment:
I'm not sure it's strictly necessary, but pdfs with taht mime-type will never hit this line if it's not there.